lib/jsdom/living/node.js

```diff
@@ -191,3 +191,25 @@ module.exports.listOfElementsWithNamespaceAndLocalName = (namespace, localName,
     } });
   });
 };
+
+// true for all node types except document ones
+function isNode(node) {
```

Comment:
Don't call it isNode then. We already have `Node.isImpl`.

Also this might not be necessary if you follow the spec.